src/Airship/Resource.hs

```diff
@@ -31,79 +31,81 @@ data PostResponse m
-             , knownContentType         :: Webmachine m Bool
+             , knownContentType     :: Webmachine m Bool
                -- | In the presence of an @If-Modified-Since@ header, returning a @Just@ value from 'lastModifed' allows
                -- the server to halt with @304 Not Modified@ if appropriate.
-             , lastModified             :: Webmachine m (Maybe UTCTime)
+             , lastModified         :: Webmachine m (Maybe UTCTime)
                -- | If an @Accept-Language@ value is present in the HTTP request, and this function returns @False@,
                -- processing will halt with @406 Not Acceptable@.
-             , languageAvailable        :: Webmachine m Bool
+             , languageAvailable    :: Webmachine m Bool
                -- | Returns @400 Bad Request@ if true. Default: false.
-             , malformedRequest         :: Webmachine m Bool
+             , malformedRequest     :: Webmachine m Bool
                                         -- wondering if this should be text,
                                         -- or some 'path' type
                -- | When processing a resource for which 'resourceExists' returned @False@, returning a @Just@ value
                -- halts with a @301 Moved Permanently@ response. The contained 'ByteString' will be added to the
                -- HTTP response under the @Location:@ header.
-             , movedPermanently         :: Webmachine m (Maybe ByteString)
+             , movedPermanently     :: Webmachine m (Maybe ByteString)
                -- | Like 'movedPermanently', except with a @307 Moved Temporarily@ response.
-             , movedTemporarily         :: Webmachine m (Maybe ByteString)
+             , movedTemporarily     :: Webmachine m (Maybe ByteString)
                -- | When handling a @PUT@ request, returning @True@ here halts processing with @300 Multiple Choices@. Default: False.
-             , multipleChoices          :: Webmachine m Bool
+             , multipleChoices      :: Webmachine m Bool
                -- | When processing a request for which 'resourceExists' returned @False@, returning @True@ here
                -- allows the 'movedPermanently' and 'movedTemporarily' functions to process the request.
-             , previouslyExisted        :: Webmachine m Bool
+             , previouslyExisted    :: Webmachine m Bool
                -- | When handling @POST@ requests, the value returned determines whether to treat the request as a @PUT@,
                -- a @PUT@ and a redirect, or a plain @POST@. See the documentation for 'PostResponse' for more information.
                -- The default implemetation returns a 'PostProcess' with an empty handler.
-             , processPost              :: Webmachine m (PostResponse m)
+             , processPost          :: Webmachine m (PostResponse m)
+               -- | As with 'processPost', but called on PATCH requests.
+             , processPatch         :: Webmachine m Bool
```

Comment:
Just curious, and this might be part of the RFC, but do you normally care about the content-type for `PATCH` in the same way you do with `PUT`?